docs/src/documentation/03-components/03-information/toast/02-react.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: React
+redirect_from:
+  - /components/toast/react/
```

Comment:
No need to have redirects for new components because they never existed on the old documentation. Remove all `redirect_from` fields for Toast documentation files. I forgot to point this out earlier in other PRs.

Reply:
I think it's existed actually. We had a page about Toast.